src/main/java/io/github/christophermanahan/carnitas/HTTPResponseBuilder.java

```diff
@@ -0,0 +1,28 @@
+package io.github.christophermanahan.carnitas;
+
+public class HTTPResponseBuilder implements ResponseBuilder {
+    private String statusCode = "";
+    private String version = "";
```

Comment:
You may want to move some of those constants here if it feels right

Reply:
I ended up moving the constants into the HTTPResponse after removing the builders!